ostd/src/mm/page/mod.rs

```diff
@@ -363,3 +330,22 @@ impl Drop for DynPage {
         }
     }
 }
+
+/// Increase the reference count of the page by one.
+///
+/// # Safety
+///
+/// The physical address must represent a valid page.
+///
+/// And the caller must ensure the metadata slot pointed through the corresponding
+/// virtual address is initialized by holding a reference count of the page firstly.
+/// Otherwise the function may add a reference count to an unused page.
```

Comment:
Not sure why this safety condition is _so complicated_. Can we simplify it like this?
```rust
/// # Safety
///
/// 1. The physical address must represent a valid page.
/// 2. The caller must have already held a reference to the page.
```

(Not a problem introduced by this PR, feel free to ignore it if you don't think it's better).